### ISSUE TITLE
Error Message on named credential

### ISSUE THREAD

**Issue by ashlynnrs** [DouglasCAyers/sfdx-mass-action-scheduler#29]
Hi, 
I am unable to complete the prerequisite steps for installing the app and am seeing the below error message when I hit save on my Named Credential. I followed instructions, and am not sure what needs fixing. Happy to provide screen shots if helpful.

Error Message:
error=redirect_uri_mismatch&error_description=redirect_uri%20must%20match%20configuration

What does it mean?

**Comment by DouglasCAyers** [comment]
Hi @ashlynnrs,

After creating the Connected App, you must [wait at least 10 minutes](https://github.com/DouglasCAyers/sfdx-mass-action-scheduler/wiki/Pre-Requisites-Instructions#3-create-connected-app) before you can move on to the next steps.

Are you still having issues if you try now? If yes, please provide screen shots of your configurations.

Thanks,

Doug

**Comment by DouglasCAyers** [comment]
Hi @ashlynnrs,

I haven't heard back from you so I'm assuming this is resolved? I'm going to close the issue, but if you're still having a problem please comment and I'll re-open.

Thanks,

Doug

**Comment by ashlynnrs** [comment]
Thanks,
To be honest I haven’t had a chance to check it out again (been out of the office a lot) but I do appreciate the quick follow-up and I will reach back out if I am still seeing issues with the credentials.

Thanks, I am really excited to try this tool out!

Ashlynn Sylvain
Data Associate
Friends General Conference
1216 Arch St. #2B
Philadelphia, Pa 19107
(215) 561-1700 x3047
ashlynns@fgcquaker.org<mailto:ashlynns@fgcquaker.org>
www.fgcquaker.org<http://www.fgcquaker.org/>

**Comment by ashlynnrs** [comment]
Doug,
I tried again today (definitely WAY over the 10 minute mark). Also started the process from scratch and attempted to work through it with a consultant, to no avail. I am still stuck and cannot get the login page after having created the named credential. I have been over this many times and cannot see where the error lies in my configuration. This is a sandbox environment, but I followed the given instructions to the letter. I have attached screen shots for my app, auth provider, and named credential, and would appreciate any light you could shed.

![auth provider](https://user-images.githubusercontent.com/38735013/39585097-fb780956-4ec1-11e8-96bf-fcd36c8d9e12.PNG)
![connected app](https://user-images.githubusercontent.com/38735013/39585109-031c8308-4ec2-11e8-9bfe-bdb2eff25571.PNG)
![named credential](https://user-images.githubusercontent.com/38735013/39585124-0c2ff344-4ec2-11e8-9128-6ae7c870015a.PNG)


Thanks again,
-Ashlynn

**Comment by DouglasCAyers** [comment]
Thanks for the screen shots @ashlynnrs. On your Auth. Provider record, what are the Callback URLs that it displays at the bottom? I imagine it may be the same as what you've copied into your Connected App's Callback URL, but I'm still curious. You also might try removing the auth provider id, 0SO3B0000004CF6, from the Callback URL `/authcallback/0SO3B0000004CF6/Mass_Action` ==> `/authcallback/Mass_Action` and see if that helps.

**Comment by ashlynnrs** [comment]
Oh my God, that did it! Thank you SO MUCH, this has been driving me nuts. I am SO EXCITED to try out this tool. Thanks for building it ☺

Ashlynn Sylvain
Data Associate
Friends General Conference
1216 Arch St. #2B
Philadelphia, Pa 19107
(215) 561-1700 x3047
ashlynns@fgcquaker.org<mailto:ashlynns@fgcquaker.org>
www.fgcquaker.org<http://www.fgcquaker.org/>

**Comment by DouglasCAyers** [comment]
Glad we got it figured out! Enjoy!

**Comment by ChrisGF** [comment]
@douglascayers I hate to bother you but I got the same error.  We haven't rolled out the custom domain as we are still in testing and my gut says that could be the issue.   I used the automated setup button to create the records.  Attached are my screenshots.  Any idea how to get by this error, your package is perfect for a problem I am having with process builder. 

![auth__providers___salesforce](https://user-images.githubusercontent.com/4898672/47799769-5248ab00-dd01-11e8-9c4c-89e03c61f0d9.jpg)

![lightning_experience___salesforce](https://user-images.githubusercontent.com/4898672/47799802-6391b780-dd01-11e8-82c3-1142e895df91.jpg)

**Comment by pcondonjr** [comment]
Doug,

I am getting the same error.  I have deployed the My Domain.  I am relatively new to SFDC and am not following much of what is in this post.  I did log out and back into environment.  It has probably been 30 minutes since the 3b. Automated Setup completed.  This is sandbox environment and I installed the App for All Users not Just Admin.  

I get the error after editing the Named Credential user (Step 4b. in Post Setup)(which already has the "Start Authentication Flow on Save " option saved), deselecting and selecting the "Start Authentication Flow on Save" and hitting save.  

Any suggestions are most welcome since I am not sure how to proceed with remaining steps in the Post Setup instructions

Thank you!
